modules/sdk/project-templates/project-templates-form-field/build.gradle

```diff
@@ -1,6 +1,6 @@
 dependencies {
 	compileOnly group: "com.beust", name: "jcommander", version: "1.72"
-	compileOnly group: "com.liferay", name: "com.liferay.project.templates.extensions", version: "1.0.15"
+	compileOnly group: "com.liferay", name: "com.liferay.project.templates.extensions", version: "1.0.16"
```

Comment:
No need to change these dependencies, I think all the needed changes can be localized to the test class.